armi/reactor/tests/test_reactors.py

```diff
@@ -200,8 +201,106 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.directoryChanger.close()
 
+    def test_kineticsParameterAssignment(self):
```

Comment:
Great test

Reply:
Might be a bit expensive, though. Would it be possible to just make an empty Core object and pass it a bunch of different settings?

Reply:
Sure thing - should be an easy swap.

Reply:
Yeah that is easy enough. The test is ~0.5 seconds without the change though.

Reply:
Moving to an empty reactor sped the test up by ~40%